### ISSUE TITLE
Validation error

### ISSUE THREAD

**Issue by pbarker** [predibase/lorax#175]
### System Info

ghcr.io/predibase/lorax:latest
MacOS 13.5
py 3.10.1

### Information

- [X] Docker
- [ ] The CLI directly

### Tasks

- [X] An officially supported command
- [ ] My own modifications

### Reproduction

from lorax import Client

endpoint_url = "https://h49afzvaswo8wa-80.proxy.runpod.net/" # running latest image

client = Client(endpoint_url)
text = client.generate("Why is the sky blue?", adapter_id="IlyaGusev/saiga_mistral_7b_lora").generated_text
print(text)

Client logs:

ValidationError: 1 validation error for Response
details.best_of_sequences
  Field required [type=missing, input_value={'finish_reason': 'length...758, 'special': False}]}, input_type=dict]
    For further information visit https://errors.pydantic.dev/2.5/v/missing
    
   
Server logs:

2024-01-11T05:10:16.760325207Z 2024-01-11T05:10:16.759939Z  INFO lorax_router: router/src/main.rs:253: Connected
2024-01-11T05:10:16.760331121Z 2024-01-11T05:10:16.759945Z  WARN lorax_router: router/src/main.rs:258: Invalid hostname, defaulting to 0.0.0.0
2024-01-11T05:13:53.611603238Z 2024-01-11T05:13:53.611172Z  INFO lorax_launcher: server.py:131 No adapter to download for base model. Skipping.
2024-01-11T05:13:53.611651257Z 
2024-01-11T05:13:53.611931900Z 2024-01-11T05:13:53.611538Z  INFO lorax_router::loader: router/src/loader.rs:150: adapter __base_model__ downloaded
2024-01-11T05:13:53.611970904Z 2024-01-11T05:13:53.611612Z  INFO lorax_router::queue: router/src/queue.rs:135: set adapter __base_model__ status to Downloaded
2024-01-11T05:13:53.613985692Z 2024-01-11T05:13:53.613542Z  INFO lorax_router::loader: router/src/loader.rs:197: adapter __base_model__ loaded
2024-01-11T05:13:53.614006769Z 2024-01-11T05:13:53.613595Z  INFO lorax_router::queue: router/src/queue.rs:135: set adapter __base_model__ status to Ready
2024-01-11T05:13:54.206549271Z 2024-01-11T05:13:54.206062Z  INFO HTTP request{otel.name=POST / http.client_ip=2601:280:5f80:1c30:857e:50a1:7ac0:2984 http.flavor=1.1 http.host=100.65.18.157:60325 http.method=POST http.route=/ http.scheme=HTTP http.target=/ http.user_agent=python-requests/2.31.0 otel.kind=server trace_id=3970ea74484441ed3d0e78a358824393}:compat_generate{default_return_full_text=Extension(false)}:generate{parameters=GenerateParameters { adapter_id: None, adapter_source: None, api_token: None, best_of: None, temperature: None, repetition_penalty: None, top_k: None, top_p: None, typical_p: None, do_sample: false, max_new_tokens: 20, return_full_text: Some(false), stop: [], truncate: None, watermark: false, details: true, decoder_input_details: false, apply_chat_template: false, seed: None } total_time="598.903007ms" validation_time="799.607µs" queue_time="5.832105ms" inference_time="592.271875ms" time_per_token="29.613593ms" seed="None"}: lorax_router::server: router/src/server.rs:424: Success
2024-01-11T05:23:37.529824123Z 2024-01-11T05:23:37.529273Z  INFO lorax_router::loader: router/src/loader.rs:150: adapter IlyaGusev/saiga_mistral_7b_lora downloaded
2024-01-11T05:23:37.529892528Z 2024-01-11T05:23:37.529345Z  INFO lorax_router::queue: router/src/queue.rs:135: set adapter IlyaGusev/saiga_mistral_7b_lora status to Downloaded
2024-01-11T05:23:37.531561517Z 2024-01-11T05:23:37.531190Z  INFO lorax_router::loader: router/src/loader.rs:241: adapter __base_model__ offloaded
2024-01-11T05:23:37.531604216Z 2024-01-11T05:23:37.531241Z  INFO lorax_router::queue: router/src/queue.rs:135: set adapter __base_model__ status to Downloaded
2024-01-11T05:23:40.459999319Z 2024-01-11T05:23:40.459681Z  INFO lorax_router::loader: router/src/loader.rs:197: adapter IlyaGusev/saiga_mistral_7b_lora loaded
2024-01-11T05:23:40.460067021Z 2024-01-11T05:23:40.459773Z  INFO lorax_router::queue: router/src/queue.rs:135: set adapter IlyaGusev/saiga_mistral_7b_lora status to Ready
2024-01-11T05:23:41.295923494Z 2024-01-11T05:23:41.295327Z  INFO HTTP request{otel.name=POST / http.client_ip=2601:280:5f80:1c30:857e:50a1:7ac0:2984 http.flavor=1.1 http.host=100.65.18.157:60325 http.method=POST http.route=/ http.scheme=HTTP http.target=/ http.user_agent=python-requests/2.31.0 otel.kind=server trace_id=7754078ad8d17c72ba4154ccd0d77f9a}:compat_generate{default_return_full_text=Extension(false)}:generate{parameters=GenerateParameters { adapter_id: Some("IlyaGusev/saiga_mistral_7b_lora"), adapter_source: None, api_token: None, best_of: None, temperature: None, repetition_penalty: None, top_k: None, top_p: None, typical_p: None, do_sample: false, max_new_tokens: 20, return_full_text: Some(false), stop: [], truncate: None, watermark: false, details: true, decoder_input_details: false, apply_chat_template: false, seed: None } total_time="7.583602179s" validation_time="408.817µs" queue_time="6.747898767s" inference_time="835.295548ms" time_per_token="41.764777ms" seed="None"}: lorax_router::server: router/src/server.rs:424: Success
    

### Expected behavior

The call to pass

**Comment by tgaddair** [comment]
Hi @pbarker, thanks for the detailed error report with repro!

I wasn't able to repro the error on my end using the script you provided. That leads me to believe the issue is a dependency difference, and since you're using our official Docker image, most likely the difference is on the client side. 

Can you share the current version of `pydantic` you're using the client?

If possible, can you try reverting to `pydantic==1.10.13` and trying again? My hunch is this is an issue with `pydantic>2`.

**Comment by pbarker** [comment]
Hey @tgaddair, thanks for getting back so quick. Yes that was indeed the issue, another library in my project was requiring pydantic >2.0

Would be nice to have compatibility in the future but this unblocks me for today, thank you!

**Comment by tgaddair** [comment]
Glad you're unblocked for now! I created #177 to track upgrading pydantic support.
